packages/salesforce/src/Adaptor.js

```diff
@@ -82,7 +82,7 @@ export function describeAll() {
 }
 
 /**
- * Outputs basic information about an sObject to `STDOUT`.
+ * Returns basic information about an sObject.
```

Comment:
`describe` and `describeAll` need to be consistent - they both have very similar signatures and descriptions.

I agree that "output to stdout" is not very useful. But `returns` isn't very accurate either.

This is one of those strange apis that doesn't write to state.data, but dumps its data in state.references.

If we're gonna make a change, I would like this (and `describeAll()`) to say: `Prints a description of an sObject and pushes the result to state.references`

Reply:
Perfect!